Cargo.toml

```diff
@@ -19,8 +19,9 @@ default = ["alloc", "p256", "x25519"]
 x25519 = ["dep:x25519-dalek"]
 p384 = ["dep:p384"]
 p256 = ["dep:p256"]
+k256 = ["dep:k256"]
 # Include serde Serialize/Deserialize impls for all relevant types
-serde_impls = ["serde", "generic-array/serde"]
+serde_impls = ["dep:serde", "generic-array/serde", "x25519-dalek?/serde", "p256?/serde", "p384?/serde", "k256?/serde", "p256?/pkcs8", "p384?/pkcs8", "k256?/pkcs8"]
```

Comment:
The `pkcs8` is possibly necessary because `Serialize` is [only impld](https://docs.rs/elliptic-curve/latest/elliptic_curve/struct.PublicKey.html#impl-Serialize-for-PublicKey%3CC%3E) for curves which are `AssociatedOid`, which is only impld when `pkcs8` is enabled.

Reply:
Yep, the serde serializers use SPKI as the serialization format for public keys, which needs the `pkcs8` feature.

I suppose it could be better documented.